modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -3,20 +3,20 @@
 // * nodes/nodes-audit-log.adoc
 
 [id="nodes-nodes-audit-log-basic-viewing_{context}"]
-= Viewing the API Audit Log
+= Viewing the audit log
 
-You can view the basic audit log.
+You can view logs for the master nodes for the {product-title} API server or the Kubernetes API server.
```

Comment:
@bergerhoffer This line is confusing me.  This makes it sound like you can view logs for each API server master node. 
Is it that the logs are from the API servers and you can view them on the OpenShift master?

Reply:
@mburke5678 this wording was taken from another module in this PR, but I agree that it's confusing.

@soltysh Looks like each master has its own log, so would it be accurate to say something like `You can view logs for the {product-title} API server or the Kubernetes API server for each master node.` And @mburke5678 do you think this is any clearer?